fontique/src/font.rs

```diff
@@ -140,27 +140,27 @@ impl FontInfo {
         &self.axes
     }
 
-    /// Returns true if the font has a `wght` axis.
+    /// Returns `true` if the font has a `wght` axis.
```

Comment:
Is there a canonical page to link to for these axes?

Reply:
My plan is to find one or find somewhere to link to that gives an indication of what you do with this info. (Suggestions welcome.)

Reply:
Found some links, added them.